projects/1.18/assets/alcocraft-beer-and-stuff/alcocraft/lang/zh_cn.json

```
@@ -0,0 +1,59 @@
+{
+  "itemGroup.alcotab": "酿酒工艺",
+
+  "block.alcocraft.spruce_keg": "酒桶",
+  "block.alcocraft.hop_block": "啤酒花植株",
+  "block.alcocraft.hop_plant_block": "啤酒花植株",
+
+  "item.alcocraft.hop": "啤酒花",
+  "item.alcocraft.hop_seeds": "啤酒花种子",
+  "item.alcocraft.dry_seeds": "干小麦种子",
+
+  "item.alcocraft.spruce_mug_empty": "空杯子",
+
+  "item.alcocraft.spruce_mug_sun_pale_ale": "太阳淡色爱尔啤酒",
+  "item.alcocraft.spruce_mug_digger_bitter": "矿工苦味啤酒",
+  "item.alcocraft.spruce_mug_nether_porter": "下界搬运工啤酒",
+  "item.alcocraft.spruce_mug_wither_stout": "凋灵世涛啤酒",
+  "item.alcocraft.spruce_mug_magnet_pilsner": "磁力皮尔森啤酒",
+  "item.alcocraft.spruce_mug_drowned_ale": "溺尸爱艾啤酒",
+  "item.alcocraft.spruce_mug_night_rauch": "夜雾啤酒",
+  "item.alcocraft.spruce_mug_ice_beer": "冰啤酒",
+  "item.alcocraft.spruce_mug_kvass": "淡啤酒",
+  "item.alcocraft.spruce_mug_leprechaun_cider": "精灵苹果酒",
+  "item.alcocraft.spruce_mug_chorus_ale": "紫颂爱艾啤酒",
+  "item.alcocraft.spruce_mug_nether_star_lager": "下界之星拉格啤酒",
+
+  "block.alcocraft.spruce_mug_block_empty": "空杯子",
+
+  "block.alcocraft.spruce_mug_block_sun_pale_ale": "太阳淡色爱尔啤酒",
+  "block.alcocraft.spruce_mug_block_digger_bitter": "矿工苦味啤酒",
+  "block.alcocraft.spruce_mug_block_nether_porter": "下界搬运工啤酒",
+  "block.alcocraft.spruce_mug_block_wither_stout": "凋灵世涛啤酒",
+  "block.alcocraft.spruce_mug_block_magnet_pilsner": "磁力皮尔森啤酒",
+  "block.alcocraft.spruce_mug_block_drowned_ale": "溺尸爱艾啤酒",
+  "block.alcocraft.spruce_mug_block_night_rauch": "夜雾啤酒",
+  "block.alcocraft.spruce_mug_block_ice_beer": "冰啤酒",
+  "block.alcocraft.spruce_mug_block_kvass": "淡啤酒",
+  "block.alcocraft.spruce_mug_block_leprechaun_cider": "精灵苹果酒",
+  "block.alcocraft.spruce_mug_block_chorus_ale": "紫颂爱艾啤酒",
+  "block.alcocraft.spruce_mug_block_nether_star_lager": "下界之星拉格啤酒",
+
+  "effect.alcocraft.flight": "飞翔",
+  "effect.alcocraft.freeze": "结霜",
+  "effect.alcocraft.phantom": "更好的休息",

```

```suggestion
  "effect.alcocraft.phantom": "休息养神",
```
原文应该是忘打中间那个连接号了，Well-rested
提供的只是建议